src/payment/strategies/mollie/mollie-payment-strategy.ts

```diff
@@ -135,6 +162,34 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
+    private async _mountCardVerificationfields(formOptions: HostedFormOptions) {
+        const { config } = this._store.getState();
+        const bigpayBaseUrl = config.getStoreConfig()?.paymentSettings.bigpayBaseUrl;
+
+        if (!bigpayBaseUrl) {
+            throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
+        }
+
+        const form = this._hostedFormFactory.create(bigpayBaseUrl, formOptions);
+
+        await form.attach();
+        this._shouldRenderHostedForm = true;
+        this._hostedForm = form;
```

Comment:
It is generally a good practice to return values out of functions. That way it is easier to follow what's going on in the caller.

```ts
this._hostedForm = await this._mountCardVerificationfields(mollie.form);
```

Reply:
Done!